app/main.py

```diff
@@ -14,10 +14,6 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :param letter: letter to find occurrences of it
     :return: count occurrences of letter in phrase
     """
-    count = 0
-
-    for char in phrase:
-        if char.lower() == letter.lower():
-            count += 1
+    count = phrase.lower().count(letter.lower())
 
     return count
```

Comment:
Good job! Could you consider to skip variable definition and put in directly into `return` statement?